pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -109,12 +109,24 @@ type ExecCommand struct {
 	LabeledCommand `json:",inline"`
 
 	// The actual command-line string
+	//
+	// Special variables that can be used:
+	//
+	//  - `$PROJECTS_ROOT`: A path path where projects sources are mounted
+	//
+	//  - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/<project-name>).  If there are multiple projects, this will point to the directory of the first one.
 	CommandLine string `json:"commandLine,omitempty"`
 
 	// Describes component to which given action relates
 	Component string `json:"component,omitempty"`
 
 	// Working directory where the command should be executed
+	//
+	// Special variables that can be used:
+	//
+	//  - `$PROJECTS_ROOT`: A path path where projects sources are mounted
+	//
+	//  - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/<project-name>).  If there are multiple projects, this will point to the directory of the first one.
```

Comment:
```suggestion
	//  - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/<project-name>). If there are multiple projects, this will point to the directory of the first one.
```